### PR TITLE
fix: apiKey was not returned from createUser mutation

### PR DESCRIPTION
**Pull request by divy-work**
One of the most silly but dangerous bugs in the history of nest.land!

@t8 I think it is time to enable 2 required reviews to merge into master.

### CODE REVIEW

File: src/db/query/user.rs [hunk 1]
```diff
@@ -40,7 +40,7 @@ pub async fn get_user_by_name(
       created_at: format!("{:?}", row.get::<usize, DateTime<Utc>>(5)),
     })
   } else {
-    Err("Not found".to_string())
+    Err("User not found".to_string())
```
Comment (t8) [inline src/db/query/user.rs:43]: Appreciate the error logging 👍

**Comment by t8** [comment]
Love these benchmarks!

**Comment by t8** [comment]
2 reviews are now required for a PR to be merged into `master`. Thanks for the reminder!

**Comment by divy-work** [comment]
Thanks!

**Pull request merged**